drush/Commands/cgov/CgovUserCommands.php

```diff
@@ -298,6 +298,10 @@ protected function loadAdditionalUser($user) {
     else {
       $pwd = NULL;
     }
+    if (array_key_exists('preferred_admin_langcode', $user)) {
```

Comment:
You should still initialize `$preferred_admin_langcode`. I would just do right above this `$preferred_admin_langcode = NULL;` to specifically set it to a value you can check for. Then lower on, instead of !empty, just `$preferred_admin_langcode !== NULL`.  (I am surprised PHPStan did not flag this... oh wait, we probably are not linting this file because it is in drush. <insert table flip gif here>)

Reply:
I have initialized $preferred_admin_langcode to NULL